bin/bb8/status.py

```diff
@@ -45,7 +46,7 @@ def last_modified_remote(fm: RemoteFileManager, paths: RemotePaths):
 
 
 def last_modified_local(target, docker_client):
-    if target.mount_id not in [v.id for v in docker_client.volumes.list()]:
```

Comment:
Maybe this should be changed to an `exists_locally` method on `Target`? Then each could have their own implementation - also good to check if the dir exists for directory targets.